### PR TITLE
ci: added docker-compose files for easier start and developing

### PR DESCRIPTION
**Pull request by tomParty**
added a couple simple docker-compose files to help spinning up and instance for local dev

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -15,3 +15,19 @@ docker build -f build/Dockerfile -t inklabs/rangedb:local .
...
 docker run -p 8080:8080 inklabs/rangedb:local
 ```
+
+## Using docker-compose to build & run locally in one step
+
+### RangeDB InMemory
+* From root dir
+    * docker-compose -f build/docker-compose.yml up --build
+* If inside build directory
+    * docker-compose up --build
```
Comment (pdt256) [inline build/README.md:25]: Stick with one strategy. I like the first: from root dir.

File: build/README.md [hunk 2]
```diff
@@ -15,3 +15,19 @@ docker build -f build/Dockerfile -t inklabs/rangedb:local .
...
+
+### RangeDB with Postgres
+* From root dir
+    * docker-compose -f build/docker-compose-pg.yml up --build
+* If inside build directory
+    * docker-compose -f build/docker-compose-pg.yml up --build
```
Comment (pdt256) [inline build/README.md:31]: Same as last comment. Remove.

**Review by pdt256 — CHANGES_REQUESTED** [review]

File: build/README.md (hunk 1 above)
Comment (tomParty) [reply build/README.md]: removed!

File: build/README.md (hunk 2 above)
Comment (tomParty) [reply build/README.md]: removed!

**Review by pdt256 — APPROVED** [review]

**Pull request merged**